help.html

```diff
@@ -1,99 +1,106 @@
-    <tr>
-        <td><b>Alt + w</b></td>
-        <td>Show the word list</td>
-    </tr>
-    <tr class="table-primary">
-        <td colspan="2">External dictionaries</td>
-    </tr>
-    <tr>
-        <td><b>Alt + 1</b></td>
-        <td>nciku</td>
-    </tr>
-    <tr>
-        <td><b>Alt + 3</b></td>
-        <td>Dict.cn</td>
-    </tr>
-    <tr>
-        <td><b>Alt + 4</b></td>
-        <td>iCIBA</td>
-    </tr>
-    <tr>
-        <td><b>Alt + 5</b></td>
-        <td>MDBG</td>
-    </tr>
-    <tr>
-        <td><b>Alt + 6</b></td>
-        <td>JuKuu</td>
-    </tr>
-    <tr>
-        <td><b>t</b></td>
-        <td>Tatoeba</td>
-    </tr>
-</table>
+        <h3>Keyboard shortcuts</h3>
+        <table class="table table-sm">
+            <tr>
+                <th class="pt-3" colspan="2">Navigation</th>
+            </tr>
+            <tr>
+                <td><kbd>N</kbd></td>
+                <td>Next word</td>
+            </tr>
+            <tr>
+                <td><kbd>B</kbd></td>
+                <td>Previous character ("back")</td>
+            </tr>
+            <tr>
+                <td><kbd>M</kbd></td>
+                <td>Next character</td>
+            </tr>
+            <tr>
+                <th class="pt-3" colspan="2">Moving the pop-up window</th>
+            </tr>
+            <tr>
+                <td><kbd>A</kbd></td>
+                <td>Move the pop-up location between three different locations</td>
+            </tr>
+            <tr>
+                <td><kbd>Y</kbd></td>
+                <td>Move pop-up window down</td>
+            </tr>
+            <tr>
+                <td><kbd>X</kbd></td>
+                <td>Move pop-up window up</td>
+            </tr>
+            <tr>
+                <th class="pt-3" colspan="2">Working with text</th>
+            </tr>
+            <tr>
+                <td><kbd>C</kbd></td>
+                <td>Copy translation to clipboard</td>
+            </tr>
+            <tr>
+                <td><kbd>Ctrl</kbd> + <kbd>C</kbd></td>
+                <td>Just copy the selected Chinese characters (without translation) to the clipboard</td>
+            </tr>
+            <tr>
+                <th class="pt-3" colspan="2">Learning words and characters</th>
+            </tr>
+            <tr>
+                <td><kbd>R</kbd></td>
+                <td>Remember word by adding it to the internal word list</td>
+            </tr>
+            <tr>
+                <td><kbd>S</kbd></td>
+                <td>Add word to Skritter queue</td>
+            </tr>
+            <tr>
+                <td><kbd>Alt</kbd> + <kbd>W</kbd></td>
+                <td>Show the internal word list</td>
+            </tr>
+            <tr>
+                <th class="pt-3" colspan="2">External dictionaries</th>
+            </tr>
+            <tr>
+                <td><kbd>Alt</kbd> + <kbd>1</kbd></td>
+                <td><a href="https://dict.naver.com/linedict/zhendict" target="_blank">LINE Dict</a></td>
```

Comment:
This link doesn't work. I think it should be this one:
https://dict.naver.com/linedict/zhendict/dict.html#/cnen/home

Reply:
It seems to work with a trailing `/`: https://dict.naver.com/linedict/zhendict/

Reply:
You're right. However, the link I'm suggesting takes you directly to the Chinese-English dictionary page.

Reply:
Both end up on the same page for me.

Reply:
Actually, they're not quite the same. You can see the difference when you delete your cookies and then go to https://dict.naver.com/linedict/zhendict/. It will take you to a page that makes you choose between English-Thai, English-Chinese, or Chinese-English. On the other hand, this link https://dict.naver.com/linedict/zhendict/#/cnen/home will always take you to the Chinese-English dictionary, even if you have deleted your cookies before.

Reply:
It's basically where the old http://nciku.com link takes you.

Reply:
I see what you mean when using private mode. Changed the link as suggested.